samples/ClassroomLabs/Modules/Library/Az.LabServices.psm1

```diff
@@ -879,7 +879,17 @@ function New-AzLab {
                 #Wait for lab to be created.
                 $lab = WaitProvisioning -uri $labUri -delaySec 60 -retryCount 120
                 #Wait for template to be provisioned.  Even labs without a template will return a environmentsetting object.
-                $defaultEnvironmentSetting = WaitProvisioning -uri $environmentSettingUri -delaySec 60 -retryCount 120 
+                $defaultEnvironmentSetting = WaitProvisioning -uri $environmentSettingUri -delaySec 60 -retryCount 120
+                
+                if ($GpuDriverEnabled) {
+
+                    # The template VM needs to be restarted for the GPU drivers to finish installing properly
+                    # This will eventually be done within the product, but for now, we need to do this explicitly
+                    Write-Verbose "Start template VM"

```

Comment:
I would specify 'Start Template VM after installing GPU drivers' otherwise people might wander why we are doing this. You can then remove the comment.